courses/migrations/0016_enrollment_order.py

```diff
@@ -0,0 +1,116 @@
+# Generated by Django 2.1.7 on 2019-06-26 16:17
+import logging
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+log = logging.getLogger()
+
+
+def backpopulate_order(apps, schema_editor):
+    """
+    Set the order for existing enrollments.
+    """
+    CourseRunEnrollment = apps.get_model("courses", "CourseRunEnrollment")
+    ProgramEnrollment = apps.get_model("courses", "ProgramEnrollment")
+    Order = apps.get_model("ecommerce", "Order")
+    Product = apps.get_model("ecommerce", "Product")
+
+    def get_order(user, product):
+        """Return the order for a user and product if any"""
+        return (
+            Order.objects.filter(lines__product_version__product=product)
+            .filter(purchaser=user)
+            .order_by("id")
+            .first()
+        )
+
+    run_enrollments = (
+        CourseRunEnrollment.objects.filter(order=None).order_by("id").iterator()
+    )
+    for enrollment in run_enrollments:
+        order = None
+        try:
+            product = Product.objects.get(
+                content_type__model="courserun", object_id=enrollment.run.id
+            )
+            order = get_order(enrollment.user, product)
+        except Product.DoesNotExist:
+            log.exception("Product does not exist for courserun %d", enrollment.run.id)
```

Comment:
Can you change this and line 50 to `log.debug` instead? I don't think the exception output is super useful and it would trigger a sentry alert on our migrations when we would really only be concerned if an order couldn't be found.